repos/system_upgrade/common/actors/checknfs/actor.py

```diff
@@ -18,6 +19,9 @@ class CheckNfs(Actor):
     tags = (ChecksPhaseTag, IPUWorkflowTag,)
 
     def process(self):
+        # if network in initramfs is enabled NFS inhibitors are redundant
+        if get_env('LEAPP_DEVEL_INITRAM_NETWORK', None):
```

Comment:
lgtm. just thinking, wouldn't be better to check specific values here? just thinking about a possibility that test could contain e.g. `LEAPP_DEVEL_INITRAM_NETWORK=0` , or typo, whatever.. I have no strong opinion here so keeping on you @fernflower

Reply:
TBH I'd rather not put any initram-network related logic into checknfs actor. Imho as far as checknfs actor is concerned if this variable is set to anything it literally says "The person calling knows what they are doing"